app/src/main/java/org/p2p/wallet/send/ui/dialogs/FreeTransactionsDetailsBottomSheet.kt

```diff
@@ -0,0 +1,45 @@
+package org.p2p.wallet.send.ui.dialogs
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.FragmentManager
+import org.p2p.uikit.utils.getColor
+import org.p2p.uikit.utils.getColorStateList
+import org.p2p.wallet.R
+import org.p2p.wallet.common.ui.bottomsheet.BaseDoneBottomSheet
+import org.p2p.wallet.databinding.DialogSendFreeTransactionsInfoBinding
+
+class FreeTransactionsDetailsBottomSheet : BaseDoneBottomSheet() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+        ) = FreeTransactionsDetailsBottomSheet().show(
+            fm, FreeTransactionsDetailsBottomSheet::javaClass.name
+        )
+    }
+
+    private lateinit var binding: DialogSendFreeTransactionsInfoBinding
+
+    override fun onCreateInnerView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        binding = DialogSendFreeTransactionsInfoBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding.itemFreeTransactions) {
+            imageViewIcon.setImageResource(R.drawable.ic_lightling)
+            textViewTitle.text = getString(R.string.free_transactions_title)
+            textViewSubtitle.text = getString(R.string.free_transactions_subtitle)
+        }
+        baseDialogBinding.buttonDone.apply {
+            text = getString(R.string.free_transactions_button)
+            backgroundTintList = getColorStateList(R.color.bg_night)
```

Comment:
why not just setBackgroundColor?

Reply:
i will check but before on OnBoarding backgroundTintList used not to broke button
maybe it was related to outline... i do not remember